### PR TITLE
[Tests] Replace `setUp()`/`tearDown()` with `before()`/`after()`

### PR DESCRIPTION
**Pull request by phansys**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | ?
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | missing yet

**TODO**:
- [x] Update the calls of `parent::setUp()` and `parent::tearDown()` where required.

See https://github.com/symfony/symfony/pull/32289#issuecomment-513384658.

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Should be done on 3.4 actually

**Comment by phansys** [comment]
Target branch was updated to `3.4`.

**Comment by nicolas-grekas** [comment]
What does it say on PHP 7.4? Maybe we should merge both PR to check, and because they are related?

File: src/Symfony/Bridge/Doctrine/Tests/ContainerAwareEventManagerTest.php [hunk 1]
```diff
@@ -20,7 +20,10 @@ class ContainerAwareEventManagerTest extends TestCase
     private $container;
     private $evm;
 
-    protected function setUp()
+    /**
+     * @before
+     */
+    protected function before(): void
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Doctrine/Tests/ContainerAwareEventManagerTest.php]: `void` cannot be used on 3.4, because PHP 5.5 is supported.

Comment (phansys) [reply src/Symfony/Bridge/Doctrine/Tests/ContainerAwareEventManagerTest.php]: You're right, I didn't remember the support for PHP 5.5.

File: src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php [hunk 2]
```diff
@@ -26,7 +26,10 @@ class DoctrineExtensionTest extends TestCase
      */
     private $extension;
 
-    protected function setUp()
+    /**
+     * @before
+     */
+    protected function before()
     {
         parent::setUp();
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php]: all similar calls should also be updated

Comment (phansys) [reply src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php]: Is `before()` available at [`TestCase`](https://github.com/sebastianbergmann/phpunit/blob/c1b8534b3730f20f58600124129197bf1183dc92/src/Framework/TestCase.php#L53)?

Comment (stof) [reply src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php]: no. But there is no parent to call at all. The parent setup will be called automatically by PHPunit (as we don't override it anymore with our own).

**Comment by nicolas-grekas** [comment]
Can you please have a look at tests? (and maybe rebase to be sure you have the latest changes)

**Comment by nicolas-grekas** [comment]
I tried to make this green, but it looks like it's almost impossible using this strategy.
I'm going to submit an alternative PR soon.
Thanks for giving it a try.

**Pull request closed without merging**